packages/tupaia-web/src/features/DashboardItem/DashboardInfoHover.tsx

```diff
@@ -3,64 +3,103 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import MuiInfoIcon from '@material-ui/icons/Info';
-import { MOBILE_BREAKPOINT } from '../../constants';
+import { Button } from '@tupaia/ui-components';
 import { Typography } from '@material-ui/core';
+import { MOBILE_BREAKPOINT } from '../../constants';
 
-const Wrapper = styled.div`
-  text-transform: none;
-  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
-  cursor: pointer;
-  color: ${({ theme }) => theme.palette.common.white};
-  border-color: ${({ theme }) => theme.palette.common.white};
-  padding: 0.3rem;
+const DesktopWrapper = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
+  color: ${({ theme }) => theme.palette.common.white};
   flex-direction: column;
-  gap: 0.3rem;
-  margin-top: 1rem;
-  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
-    .MuiButton-label {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      align-items: center;
-    }
-    position: absolute;
-    width: 100%;
-    height: 100%;
+  position: absolute;
+  padding: 1rem;
+  width: 100%;
+  height: 100%;
+  background-color: ${({ theme }) => theme.palette.common.black};
+  top: 0;
+  left: 0;
+  opacity: 0;
+
+  svg {
+    font-size: 1.4rem;
+  }
+
+  p {
+    font-size: 0.9rem;
+  }
+
+  &:hover,
+  &:focus-visible {
+    cursor: auto;
+    opacity: 0.9;
     background-color: ${({ theme }) => theme.palette.common.black};
-    border: none;
-    top: 0;
-    left: 0;
-    opacity: 0;
-    margin-top: 0;
-    &:hover,
-    &:focus-visible {
-      opacity: 0.9;
-      background-color: ${({ theme }) => theme.palette.common.black};
-    }
+  }
+
+  @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
+    display: none;
   }
 `;
 
-const DashboardInfoHoverText = styled(Typography)`
-  font-size: 1rem;
+const MobileWrapper = styled.div`
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    display: none;
+  }
+`;
+
+const ExpandButton = styled(Button).attrs({
+  variant: 'outlined',
+  color: 'default',
+})<{
+  $active: boolean;
+}>`
+  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
+  text-transform: none;
+  color: ${({ theme }) => theme.palette.common.white};
```

Comment:
I wonder if we should be using the primary text (etc) variables here on the off change things change